segmentation/utils/synthetic_spatial_datagen.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+import xarray as xr
+import pandas as pd
+import skimage.measure
+import scipy
+import os
+from scipy.spatial.distance import cdist
+
+
+# Constants for random distance matrix generation
+AB_DIST_MEAN = 100
+AB_DIST_VAR = 1
+
+AC_DIST_MEAN = 20
+AC_DIST_VAR = 1
+
+
+# Constants for random centroid matrix generation
+A_CENTROID_FACTOR = 0.5
+B_CENTROID_FACTOR = 0.9
+C_CENTROID_FACTOR = 0.4
+
+A_CENTROID_COV = [[1, 0], [0, 1]]
+B_CENTROID_COV = [[1, 0], [0, 1]]
+C_CENTROID_COV = [[1, 0], [0, 1]]
+
+
+def direct_init_dist_matrix(num_A=100, num_B=100, num_C=100, distr_AB=None, distr_AC=None, seed=None):
```

Comment:
Depending on whether this is used for development-testing or by lab users, it might be good to set the default seed to 0.  If it's for development/testing, we'll probably want to default to a set seed, and set `seed=None` manually.  If it's for lab users, the current default is fine, since they probably want 'non-seeded' randomization.

Honestly, it depends on which behavior is used more often; I don't know the full context of this function's intended usage

Reply:
I'll have to talk to Noah about this, thanks for pointing it out.